.gitignore

```diff
@@ -0,0 +1,44 @@
+# Prerequisites
+*.d
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
+*.smod
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+*.lib
+
+# Executables
+*.exe
+*.out
+*.app
```

Comment:
Are all these necessary?

Reply:
Nope, not necessary. It was there simply for convenience. I use git status a lot and seeing a clean status makes me feel good :). I can remove and push if you'd like.

Reply:
Oh just caught where you put the message. And no they're not! I got this ignore from the standard cpp ignore file github generates and then added some myself. I'lll go ahead and remove these.

Reply:
Fixed.